[Savage Worlds Tabbed 3.7] Select skills available for current Setting.

gronyon opened the pull request:
## Changes / Comments
- [Savage Worlds Tabbed 3.7]
- Added skill selection allowed in current setting
- Enabled extra i18n entries
- Fixed Wounds/Fatigue modifiers placeholders and i18n issues
- Fixed Wounds/Fatigue CSS in "long label languages" like French

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [x] Does the pull request title have the sheet name(s)? Include each sheet name.
- [x] Is this a bug fix?
- [x] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

clevett commented:
Greetings @gronyon 

It looks like a merge has resulted in conflicts. The merge conflicts will  need to be resolve before I can merge this pr.

Thanks,
Cassie

gronyon commented:
Hello,
Thank you for the information, I didn't see I had conflits.
I'll have to find help from friends, because I'm never had to manage
conflict with Git before, but I'll work on it.

See you soon,
Cyril aka Gronyon


Le mar. 13 août 2019 à 02:56, Cassie Levett <notifications@github.com> a
écrit :

> Greetings @gronyon <https://github.com/gronyon>
>
> It looks like a merge has resulted in conflicts. The merge conflicts will
> need to be resolve before I can merge this pr.
>
> Thanks,
> Cassie
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Roll20/roll20-character-sheets/pull/5364?email_source=notifications&email_token=AJCZDUVDWML4MFLLL6E2PILQEIBFBA5CNFSM4IJFRFMKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD4EHLYI#issuecomment-520648161>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJCZDUUKHH2R6IVB24VCZGTQEIBFBANCNFSM4IJFRFMA>
> .
>

gronyon commented:
Hi @clevett ,

I have fixed conflicts and polished the integration of my changes into Finderski's.
I also added French translations to Finderski's changes.

Everything shall be ok.
